hackathon_site/dashboard/frontend/src/pages/Inventory/Inventory.tsx

```diff
@@ -142,8 +160,13 @@ const Inventory = () => {
                                 size="large"
                                 fullWidth={true}
                                 disableElevation
+                                onClick={getMoreHardware}
                             >
-                                Load more
+                                {isMoreLoading ? (
+                                    <CircularProgress size={25} />
```

Comment:
Can you tweak the test to also check that this appears, and then disappears once the data is loaded? You'll need to add a test id to it.

That becomes our integration-y test that the `isMoreLoading` selector is actually working. I do something similar here: https://github.com/ieeeuoft/hackathon-template/blob/develop/hackathon_site/dashboard/frontend/src/components/inventory/InventoryFilter/InventoryFilter.test.tsx#L157